resources/assets/lib/utils/beatmapset-discussion-helper.ts

```diff
@@ -29,7 +29,7 @@ export function propsFromHref(href: string) {
 
   if (targetUrl.host === window.location.host) {
     const target = BeatmapDiscussionHelper.urlParse(targetUrl.href, null, { forceDiscussionId: true });
-    if (target?.discussionId != null) {
+    if (target?.discussionId != null && target?.beatmapsetId != null) {
```

Comment:
the typing for the function being wrong aside, the second `?` is not needed because it's already guaranteeed by the first one?